clang/include/clang/Sema/Sema.h

```diff
@@ -1162,6 +1162,11 @@ class Sema final {
   /// CurContext - This is the current declaration context of parsing.
   DeclContext *CurContext;
 
+  SemaOpenACC &OpenACC() {
```

Comment:
nit: per style guide, this should be `openACC()` (lowercase O), and it should be `getOpenAcc()` (verb phrase).

Personally, I think the verb-phrase rule when applied to getters is harmful enough to be worth ignoring (preferring [name according to side-effects](https://www.swift.org/documentation/api-design-guidelines/#name-according-to-side-effects)). So I'd be happiest with `openACC()`, but also fine with `getOpenACC()`.

(Picking this nit because I suspect we're going to have a bunch of these, and they should be consistent.)

Reply:
I think @sam-mccall is reading the coding guidelines the same way I read them, and I agree with his conclusion that I also prefer `openACC()` over `getOpenACC()`. I don't find `get` to add much value to readability in this case and I think deviating from the style guideline is appropriate.